specs/2025-02-24.yaml

```diff
@@ -2594,6 +2750,113 @@ components:
         state:
           type: string
           description: State, county, province, or region
+    OrganizationRequest:
+      type: object
+      description: |-
+        Represents a request for a nonprofit organization to join Chariot. Grantmakers create these requests when they want to send disbursements to nonprofits that don't yet exist in Chariot's system.
+      required:
+        - id
+        - ein
+        - organization_name
+        - created_at
+        - updated_at
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the onboarding request
+          example: "550e8400-e29b-41d4-a716-446655440000"
+          readOnly: true
+        ein:
+          type: string
+          description: The Employer Identification Number (EIN) of the nonprofit organization
+          example: "123456789"
+        referring_organization_id:
```

Comment:
why do we need to expose this? can grantmaker A see that grantmaker B requested nonprofit C? should be no.. and that grantmakers don't see other requests from other grantmakers